sim/priest/talents.go

```diff
@@ -68,7 +68,7 @@ func (priest *Priest) applyForceOfWill() {
 
 	priest.OnSpellRegistered(func(spell *core.Spell) {
 		if spell.Flags.Matches(SpellFlagPriest) {
-			spell.DamageMultiplier *= .01 * float64(priest.Talents.ForceOfWill)
+			spell.DamageMultiplier *= 1.01 * float64(priest.Talents.ForceOfWill)
```

Comment:
These talents should actually be applied to the additive multiplier because they use the `Modifies Damage/Healing Done
` effect https://www.wowhead.com/classic/spell=18550/force-of-will but I can do a fast follow to update them because I'm sure a bunch are wrong in the classic code from being dated